### PR TITLE
Manifests

### PR DESCRIPTION
**Pull request by josefkarasek**

### CODE REVIEW

File: Dockerfile.rhel7 [hunk 1]
```diff
@@ -0,0 +1,14 @@
+FROM registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.17-openshift-4.10 AS builder
```
Comment (sherine-k) [inline Dockerfile.rhel7]: we still can do the build on rhel-8 and copy build result to rhel7 image I'm assuming?

File: Makefile [hunk 2]
```diff
@@ -16,15 +16,25 @@ include $(addprefix ./vendor/github.com/openshift/build-machinery-go/make/, \
...
-	$(CONTAINER_ENGINE) push ${TARGET_REPO}/${IMG}:$(IMAGE_TAG)
\ No newline at end of file
+	$(CONTAINER_ENGINE) push ${IMG}:$(IMAGE_TAG)
+
+.PHONY: push.image.rhel7
+push.image.rhel7: build.image.rhel7
+	$(CONTAINER_ENGINE) push ${IMG}:${IMAGE_TAG}
+
+deploy: push.image.rhel7
+	oc new-project node-observability-operator
```
Comment (sherine-k) [inline Makefile:39]: so we're going single namespace (operator and operand)?

Comment (josefkarasek) [reply Makefile:39]: This is for local development. The operator creates these resources in production.

File: Dockerfile.rhel7 (hunk 1 above)
Comment (josefkarasek) [reply Dockerfile.rhel7]: that was a copy paste error, `registry.ci.openshift.org/ocp/4.10:base` is rhel8 based

**Review by sherine-k — APPROVED** [review]

**Pull request merged**